opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/consent/Xs2aConsentInfo.java

```diff
@@ -34,6 +34,13 @@ public boolean isMultipleScaAvailable(Xs2aContext ctx) {
         return null != ctx.getAvailableSca() && !ctx.getAvailableSca().isEmpty();
     }
 
+    /**
+     * Is the current consent authorization using zero SCA flow
+     */
+    public boolean isZeroScaAvailable(Xs2aContext ctx) {
```

Comment:
Looks like you have forgotten to add `isZeroScaAvailable` to `Zero SCA` sequence both for payment and for consent

Reply:
Function was added in both places